### ISSUE TITLE
Test failure in stats using numpy 1.10.0b1

### ISSUE THREAD

**Issue by WarrenWeckesser** [scipy/scipy#5148]
This is on a Macbook (OS X version 10.9.5):
```
$ python -c "import scipy.stats as m; m.test('full')"
Running unit tests for scipy.stats
NumPy version 1.10.0b1
NumPy relaxed strides checking option: True
NumPy is installed in /Users/warren/local_scipy/lib/python2.7/site-packages/numpy
SciPy version 0.17.0.dev0+5c8b9ab
SciPy is installed in /Users/warren/local_scipy/lib/python2.7/site-packages/scipy
Python version 2.7.10 |Anaconda 1.8.0 (x86_64)| (default, May 28 2015, 17:04:42) [GCC 4.2.1 (Apple Inc. build 5577)]
nose version 1.3.6
.........................................................................................................................................................................................................................................................................................................................................................................................................................................................................................................................................................................................................................................................................................................................................................................................................................................................................................................................................................K............K..........................................................................................................................................................S...........S.............................................................................................................................................................................................................................................................................................................................................................................................................................................KKKKK.......................................................................................KKKK.....................................................................................................................................................KKKKK....................................................................................................................................................................................................................................................................................................................K.................................................K............................S.................................................................S...................................................................................................................................S.........................................................................................................................................................................................................K..K...............KK.KK..................K....K......K.K.K..K.K.........K..K.K....K.................................................................................................................................................................S..S........................................................................................................................................................................................................................................................F...............................................................................................................................................................................................................................................
======================================================================
FAIL: test_weighting (test_stats.TestHistogram)
----------------------------------------------------------------------
Traceback (most recent call last):
  File "/Users/warren/local_scipy/lib/python2.7/site-packages/scipy/stats/tests/test_stats.py", line 923, in test_weighting
    decimal=2)
  File "/Users/warren/local_scipy/lib/python2.7/site-packages/numpy/testing/utils.py", line 886, in assert_array_almost_equal
    precision=decimal)
  File "/Users/warren/local_scipy/lib/python2.7/site-packages/numpy/testing/utils.py", line 708, in assert_array_compare
    raise AssertionError(msg)
AssertionError: 
Arrays are not almost equal to 2 decimals

(mismatch 40.0%)
 x: array([   4. ,    0. ,    4.5,   -0.9,    0. ,    0.3,  110.2,    0. ,
          0. ,   42. ])
 y: array([   4. ,    0. ,    4.5,   -0.9,    0.3,    0. ,    7. ,  103.2,
          0. ,   42. ])

----------------------------------------------------------------------
Ran 3305 tests in 78.886s

FAILED (KNOWNFAIL=35, SKIP=7, failures=1)
```

**Comment by jaimefrio** [comment]
This is related to [this](https://github.com/numpy/numpy/issues/6120). The summary is that it is the test that needs fixing, probably by removing it, as it is testing to what bin do floating point values lying on a bin boundary should be assigned.

**Comment by rgommers** [comment]
duplicate of gh-5173 and fixed in 0.16.1, so closing. Actually this bug report is older, but the other one is referenced in the fix and the release notes. So marking this as duplicate.
